multigeiger/multigeiger.ino

```diff
@@ -713,24 +707,20 @@ void DisplayStartscreen(void){
   u8x8.drawString(0, 2, "Geiger-");
   u8x8.drawString(0, 3, " Counter");
   char rv[9];
-  char delim[] = "_-";
-  char *p; 
-  p = strtok((char *)rvStr,delim); 
-  strcpy(rv, p);
+  sprintf(rv,"V%s",software_version);
   u8x8.drawString(2, 4, rv);
   *rv = '\0';
-  p = strtok(NULL,delim);
+  strcpy(softversdat, software_version_date);
```

Comment:
sometimes you leave a space after comma (which is good, better readable), but often not.

would be greatly improve readability if you always did.

Reply:
and can we please keep the "e", so have "date", not "dat"?